packages/react/src/components/Accordion/__tests__/__snapshots__/Accordion-test.js.snap

```diff
@@ -48,12 +46,11 @@ exports[`Accordion should render 1`] = `
                 xmlns="http://www.w3.org/2000/svg"
               >
                 <svg
-                  aria-label="Expand/Collapse"
+                  aria-hidden={true}
```

Comment:
Just curious as to why all these have `aria-hidden={true}` now, don't see any related changes 🤔

Reply:
@tw15egan I think it's because by default our SVG icons will have `aria-hidden` set if no `aria-label` is provided 👀